TeaLeaf/src/com/tealeaf/TeaLeaf.java

```diff
@@ -507,7 +507,12 @@ public void onWindowFocusChanged(boolean hasFocus) {
 			/* Fix Crash in Android 7.0 Samsung devices where screenOffReceiver is null
 			TODO: Need to find why it is null */
 			if (screenOffReceiver != null) {
-				unregisterReceiver(screenOffReceiver);
+				try {
+					unregisterReceiver(screenOffReceiver);
+				} catch (Exception e) {
+					logger.log("{focus} Unable to unregister screenOffReciever");
+					e.printStackTrace();
```

Comment:
Why are we doing this ?

Reply:
There are cases where reciever is already been unregistered,In that case crash was happening.It's safe to put try/catch here.

Reply:
https://stackoverflow.com/a/9950673/2911650

Reply:
I think try catch is not needed here. But won't cause any harm.

Reply:
We can remove it as we are already checking for null value.